justaddcl/yujinelson.com
# [CU-2x1wjpj] Update about page for 2022

## Background
Some copy on the about page is outdate and doesn't reflect the role at Duffel, the move to the UK, or other updates on my CV. I need this updated to accurately reflect my location and work experience and be consistent with what I have on my CV.

Also, as I was recently laid-off, I should be clear that Duffel is not my current role, but my most recent role. 

## Updates
- Added Duffel role bullet points to experience data
- Updated copy to reflect updated location and title
- Updates skills to be more consistent with CV
- Updated hobbies blurb
- Changed "Current role" to "My most recent role"
- Updated the `ExperienceCard` to show the role at Duffel
- Added 'formik', 'meilisearch', and 'programmes', to `spellcheck` list

## How to test
- Ensure the copy updates are reflected
- Ensure the `ExperienceCard` shows my Duffel role, not Magpul
- Ensure the updates to the skills section are reflected
